seatunnel-dist/pom.xml

```diff
@@ -677,7 +677,7 @@
                     <version>${project.version}</version>
                     <scope>provided</scope>
                 </dependency>
-                
+
```

Comment:
Please add slack dependency to seatunnel profile. Around 200 to 300 lines

Reply:
I have added it! PTAL